app/models.py

```diff
@@ -388,6 +391,17 @@ def serialize(self, data=None):
             "countersignedAt": countersigned_at,
         }, **(data or {}))
 
+        if self.agreement_details and self.agreement_details.get('uploaderUserId'):
+            user = User.query.filter(
+                User.id == self.agreement_details.get('uploaderUserId')
+            ).first()
+
+            if user:
```

Comment:
@laurenceberry has said that we should just be doing the ignoring thing.

If you're considering the CSS admin user looking at a contract page, the choices seem to be:
- the frontend app fails with a 500 error after not finding the user's information
- there's a blank space under "Uploader name & email" in the admin app, where normally we would see a name and email.

It seems like, on the very outside chance that a user's ID was saved here but then their account gets deleted or something, that at least CCS could get some useful information out of the contract page rather than just having everything break.